codegen/builtin_option.go

```diff
@@ -586,14 +590,15 @@ func (f Forward) Call(ctx context.Context, cln *client.Client, ret Register, opt
 		}
 
 		localPath = filepath.Join(dir, "proxy.sock")
-		id = digest.FromString(localPath).String()
+		id = digest.FromString(src.String()).String()
```

Comment:
This was causing cache invalidation every time we used the `forward` with `tcp` since the proxy.sock file was always in a new tmp directory.